src/introduction.md

```diff
@@ -9,4 +9,8 @@ In this guide, we'll discuss:
 * What each edition is about
 * How to migrate your code from one edition to another
 
-Read on for more!
\ No newline at end of file
+Note that the standard library grows with each Rust release; there are *many*
+additions to the standard library that are not called out in this guide. Only
+the most absolutely major ones are, but there's tons of medium and small
```

Comment:
most absolutely major ones? wow, they must be real special! 😜 
I think "major ones" suffices :)